components/Faq.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
If accordion and faq are not reusable components I suggest to keep the functionality inside the faq tsx page.

Reply:
I made it like that because in future it can become reusable components. If any other page needs this accordion and also we can add the question answer by passing category as props so it can be reused if we have some other categories of questions like we are doing with "general"